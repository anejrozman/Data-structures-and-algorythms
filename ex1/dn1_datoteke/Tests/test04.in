65 51 54 89 39 29 96 63 49 78 84 3 52 38 82 85 32 7 50 88 87 9 74 99 61 8 100 45 37 25 53 16 6 46 5 77 81 34 79 67 33 80 42 4 43 2 48 59 19 90 47 21 76 93 22 14 91 17 31 83 10 18 27 86 70 41 57 12 95 44 64 97 30 15 55 69 28 40 20 11 24 75 23 36 13 68 92 56 94 73 58 98 60 72 26 71 1 35 66 62
