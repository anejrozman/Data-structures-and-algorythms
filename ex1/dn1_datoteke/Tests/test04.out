None 0 0 None 3 4 None 6 7 6 6 10 10 12 10 6 15 16 15 6 19 20 20 None 23 24 None 26 27 28 26 30 31 30 33 26 26 36 36 38 39 36 41 42 41 44 41 41 47 26 49 50 49 26 53 54 53 56 56 56 59 59 59 56 63 64 64 66 26 68 68 26 71 72 71 71 75 75 77 78 77 71 81 81 83 81 71 86 71 88 89 26 91 91 93 93 95 95 95 98
3 2 3 6 6 6 23 9 9 10 15 12 14 14 15 19 18 18 19 23 23 22 23 26 26 26 None 30 30 30 35 33 33 35 35 36 49 38 41 41 41 49 44 44 46 46 47 49 49 53 52 52 53 68 56 56 68 58 59 63 61 62 63 68 68 66 68 68 71 70 71 91 74 74 75 81 77 81 80 80 81 86 83 85 85 86 88 88 91 91 91 None 93 None 95 None 97 98 None None
